### PR TITLE
Fixed some incorrect labels and added some e2e.

### PR DESCRIPTION
**Pull request by ty-dc**
#### What type of PR is this?

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/workflows/e2e-init.yaml [hunk 1]
```diff
@@ -141,6 +141,7 @@ jobs:
               echo "The current kubernetes version is ${{ inputs.k8s_version }} , ignore to install openvswitch due to Kind base image outdated"
               INSTALL_OVS_VALUE=false;
             fi
+            echo "INSTALL_OVS=${INSTALL_OVS_VALUE}" >> $GITHUB_ENV
```
Comment (Icarus9913) [inline .github/workflows/e2e-init.yaml:144]: no need

File: .github/workflows/e2e-init.yaml [hunk 2]
```diff
@@ -165,6 +166,7 @@ jobs:
           make ${{ matrix.e2e_test_mode }} -e E2E_CLUSTER_NAME=${{ env.E2E_CLUSTER_NAME }}  \
               -e E2E_GINKGO_LABELS=${E2E_LABELS} \
               -e E2E_TIMEOUT=${{ env.E2E_TIME_OUT }} \
+              -e INSTALL_OVS=${{ env.INSTALL_OVS }} \
```
Comment (Icarus9913) [inline .github/workflows/e2e-init.yaml:169]: use the E2E codes to check whether we have ovs SMC or IPPool resource

Comment (ty-dc) [reply .github/workflows/e2e-init.yaml]: I don't need ovs, I just need to confirm that the node has an additional network card for network card abnormality testing, such as eth1.

But you're right, I shouldn't judge in CI, put it in It.

**Review by weizhoublue — APPROVED** [review]

**Review by Icarus9913 — APPROVED** [review]

**Pull request merged**